### ISSUE TITLE
Enhanced function

### ISSUE THREAD

**Issue by DDXDB** [lithium0003/bunkoOCR-windows#1]
Can you specify the output path and combine multiple json results into a single txt file?
Can you develop a localization interface so that others can localize the program?
can you add compilations without testdata in Releases to make updates faster?
Thanks to DirectML, my ARC A770 GPU is running just fine!

**Comment by lithium0003** [comment]
Thanks for suggestions. This app is now just run ML model, no satisfied interfaces, just start point.
Now, improving ML model for more detection accuracy. Next, interface functions should be refined.
Please wait it.

**Comment by DDXDB** [comment]
> Thanks for suggestions. This app is now just run ML model, no satisfied interfaces, just start point. Now, improving ML model for more detection accuracy. Next, interface functions should be refined. Please wait it.

No, no, no, I mean can we localize Form. So that the program UI supports other languages (such as English)

**Comment by lithium0003** [comment]
Oh, Form UI is now just constructing for a release. Next release, make UI localizable.
If you can support other languages, welcome to contribute.

**Comment by lithium0003** [comment]
In new version, UI update and written in English.
Now, output folder can be specified.
